### DISCUSSION TITLE
Change "Playing (Music Player)" to something custom

### DISCUSSION THREAD

**Discussion by SaqinNoor** [jojo2357/Music-Discord-Rich-Presence#75] [Q&A] [answered]
Hi, is there a way to change "Playing Music Bee" to something custom like "Playing Music"? Tried to look for it in DiscordPresenceConfig.ini but couldn't find it :/

**Comment by Smaltin** [comment, marked as answer]
Whatever you set the name of your Discord application to (for custom presence) will be the name of the presence, so if you name it "Music" then it'll appear as "playing music." You can create a new default for whatever application you're using and just name it "Music" if you wish to overwrite it.
